securedrop_client/app.py

```diff
@@ -70,7 +74,16 @@ def configure_logging():
     sys.excepthook = excepthook
 
 
-def run():
+def arg_parser() -> ArgumentParser:
+    parser = ArgumentParser('securedrop-client',
+                            description='SecureDrop Journalist GUI')
+    parser.add_argument(
+        '-H', '--home', default=DEFAULT_HOME,
+        help='Home directory for storing files and state')
```

Comment:
Ok, then I propose we call it `sdc_home` or `client_home` to mimic `--gpg-home`. I think `data_dir` isn't totally accurate because to me logs are configs aren't "data" in the `/var/lib` sense that the term "data dir" often gets used with.

Reply:
I'm cool with either of these names tbh